starWarsScene.js

```diff
@@ -1,8 +1,9 @@
 function spotEachRebel(rebels) {
   var spottedRebelCalls = []
-
-  if (rebels) {
+  var i = 0
+  while (rebels[i]) {
```

Comment:
Nicely done. Your other option would have been a for loop. Both are correct approaches.

```
for(var i = 0; i < rebels.length; i++) {
  spottedRebelCalls.push('Look! Rebel scum!')
}
```